Add WebDriver conformance section for 116 release notes (#28298)
mdn/content: files/en-us/mozilla/firefox/releases/116/index.md

#### WebDriver BiDi

> - Added support for the `session.end` command that allows users to terminate the automation session ([Firefox bug 1829337](https://bugzil.la/1829337)). This was previously only possible for sessions using both WebDriver Classic and WebDriver BiDi. It is now possible also for WebDriver BiDi-only sessions.
> - Added support for [capability matching](/en-US/docs/Web/WebDriver/Capabilities) for the `session.new` command ([Firefox bug 1731730](https://bugzil.la/1731730)). It allows to define expectations about the target browser, such as browser name, platform name, … and can also used to configure the session (for example, specifiy if insecure certificates should be accepted).

Reviewer: Nit: the convention is to put the bugzilla link at the end:

```suggestion
- Added support for the `session.end` command that allows users to terminate the automation session. This was previously only possible for sessions using both WebDriver Classic and WebDriver BiDi. It is now possible also for WebDriver BiDi-only sessions ([Firefox bug 1829337](https://bugzil.la/1829337)).
- Added support for [capability matching](/en-US/docs/Web/WebDriver/Capabilities) for the `session.new` command. It allows to define expectations about the target browser, such as browser name, platform name, etc. It can also be used to configure the session, for example, to specify if insecure certificates should be accepted ([Firefox bug 1731730](https://bugzil.la/1731730)).
```

- Shadow roots are now correctly serialized when they are the root of a returned value ([Firefox bug 1836514](https://bugzil.la/1836514)).
- The `network` event time origin information was renamed from `originTime` to `timeOrigin` ([Firefox bug 1836926](https://bugzil.la/1836926)).
- The `network` event `network.responseCompleted` is now correctly emitted for navigation requests involving a redirect ([Firefox bug 1838238](https://bugzil.la/1838238)).

#### Marionette

> - Removed support for the `moz:useNonSpecCompliantPointerOrigin` capability ([Firefox bug 1490258](https://bugzil.la/1490258)). Users who still need this feature can still use the latest ESR release, Firefox ESR 115.

Reviewer: Same nit:

```suggestion
- Removed support for the `moz:useNonSpecCompliantPointerOrigin` capability. Users who still need this feature can still use the latest ESR release, Firefox ESR 115 ([Firefox bug 1490258](https://bugzil.la/1490258)).
```

> - Removed support for the `moz:useNonSpecCompliantPointerOrigin` capability. Users who still need this feature can still use the latest ESR release, Firefox ESR 115 ([Firefox bug 1490258](https://bugzil.la/1490258)).

Reviewer 2: Maybe we can just say `can still use the Firefox 115 ESR release as long as it is supported`? That would prevent the need to have it called out twice. 

Also it might be good to give a hint that a bug should be filed under [Remote Protocol :: Marionette](https://bugzilla.mozilla.org/enter_bug.cgi?product=Remote%20Protocol&component=Marionette) in case there is an issue.

Author: sure, I tweaked the sentence and added one for filing bugs

> - A regression was fixed that prevented us from differentiating stale elements (DOM elements that have been seen before on the page) from unknown elements for a given browsing context ([Firefox bug 1822466](https://bugzil.la/1822466))

Reviewer: Missing punctuation
```suggestion
- A regression was fixed that prevented us from differentiating stale elements (DOM elements that have been seen before on the page) from unknown elements for a given browsing context ([Firefox bug 1822466](https://bugzil.la/1822466)).
```
